pipelines/validator/generate-data.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# [wf] execute generate-data.sh stage
+
+# [wf] Checks if statsmodels module is installed.
+if python -c "import statsmodels.stats.api" &> /dev/null; then
+    echo 'requirements already installed.'
+else
+    echo 'Installing dependencies'
+    pip install statsmodels
+    pip install scipy
```

Comment:
would be better to create a virtual environment or use a container. Installing directly to the host might cause failures since `pip` directly on a host usually requires admin privileges

Reply:
Is it okay if I just make sure that a virtual env is active?

Reply:
that would work but might be unexpected for people that run `popper run` in an already-active virtual env. So maybe first checking if one is active, and if it is, de-activating and creating one? We could name it something like `popper-run-validator-pipeline` or something descriptive like this, so that they know where the env was created/activated from